Formula/libparc.rb

```diff
@@ -0,0 +1,23 @@
+class Libparc < Formula
+  desc "This library is part of hICN stack"
+  homepage "https://github.com/FDio/cicn/blob/cframework/master/libparc/README.md"
+  url "https://github.com/FDio/cicn", :using=>:git, :branch=>"cframework/master"
+  version "1.0-65~g7944543~b3"
+
+  bottle do
+    sha256 "d915407ef1a9a22321a8fc4e8a4186b305d901b31858136cfa4d7f885c2bb0b9" => :mojave
+  end
+
+  depends_on "cmake" => :build
+  depends_on "libevent"
+  depends_on "openssl@1.1"
+
+  def install
+    system "cmake", "libparc", *std_cmake_args
+    system "make", "install", "-j"
+  end
+
+  test do
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.